HIP/hip-632.md

```diff
@@ -132,6 +170,11 @@ Give example of using each function in context to show how they can be used to d
 ## Reference Implementation
 
 
+## (New) Response Codes That Can Be Returned
+
+* `INVALID_ARGUMENT_TO_SYSTEM_CONTRACT_METHOD` - an argument to a system contract method is invalid (e.g., byte array of wrong length)
```

Comment:
Have we checked that there are not existing response codes that could be used for these purposes.  Just wondering as adding response codes I believe at entails updating the HederaResponseCodes.sol file at the very least.

Reply:
I did check, didn't find anything that looked appropriate.